images/daos-client-image.json

```diff
@@ -41,14 +41,12 @@
     {
       "type": "shell",
       "execute_command": "echo 'packer' | sudo -S env {{ .Vars }} {{ .Path }}",
-      "environment_vars": "IO500_INSTALL_DIR={{user `IO500_INSTALL_DIR`}}",
-      "script": "./scripts/mfu_install.sh"
+      "script": "./scripts/mpifileutils_install.sh"
     },
     {
       "type": "shell",
       "execute_command": "echo 'packer' | sudo -S env {{ .Vars }} {{ .Path }}",
-      "environment_vars": "IO500_INSTALL_DIR={{user `IO500_INSTALL_DIR`}}",
-      "script": "./scripts/io500_install.sh"
+      "script": "./scripts/io500-sc21_install.sh"
```

Comment:
I wondered about this myself.  

My preference is to not to include the IO500 install in the client images.  It seems like something that should be done by a startup script or an admin after the instances are up. 

**Possible options:**
- startup script
- move that IO500 install script to terraform/examples/io500/ and rename it install_io500-sc21.sh. Have the terraform/examples/io500/start.sh script run install_io500-sc21.sh after the instances are up. 
- move the contents of the current io500-sc21_install.sh script into the terraform/examples/io500/run_io500-sc.sh file so that it would install io500-sc21 if it hasn't already been installed and then run it.

Reply:
I like bullets 2 or 3 (very similar)

Reply:
Went with the option listed in bullet 2 above.